phpBB/styles/prosilver/template/navbar_footer.html

```diff
@@ -4,18 +4,10 @@
 	<ul id="nav-footer" class="nav-footer linklist" role="menubar">
 		<li class="breadcrumbs">
 			<!-- IF U_SITE_HOME -->
-				<span class="crumb">
-					<a href="{U_SITE_HOME}" data-navbar-reference="home">
-						<i class="icon fa-home fa-fw" aria-hidden="true"></i><span>{L_SITE_HOME}</span>
-					</a>
-				</span>
+				<span class="crumb"><a href="{U_SITE_HOME}" data-navbar-reference="home"><i class="icon fa-home fa-fw" aria-hidden="true"></i><span>{L_SITE_HOME}</span></a></span>
 			<!-- ENDIF -->
 			<!-- EVENT overall_footer_breadcrumb_prepend -->
-			<span class="crumb">
-				<a href="{U_INDEX}" data-navbar-reference="index">
-					<!-- IF not U_SITE_HOME --><i class="icon fa-home fa-fw" aria-hidden="true"></i><!-- ENDIF --><span>{L_INDEX}</span>
-				</a>
-			</span>
+				<span class="crumb"><a href="{U_INDEX}" accesskey="h" data-navbar-reference="index"><!-- IF not U_SITE_HOME --><i class="icon fa-home fa-fw"></i><!-- ENDIF --><span>{L_INDEX}</span></a></span>
```

Comment:
`accesskey="h"` on the `<a>` tag is new and `aria-hidden="true"` is on the `li` tag